src/tools/m65common.c

```diff
@@ -335,6 +342,7 @@ int stop_cpu(void)
   fprintf(stderr, "Stopping CPU\n");
   if (no_rxbuff)
     do_usleep(50000);
```

Comment:
Hier scheint es mir, dass deine Änderung richtig sein soll.  Aber eine Errinerung nervt mir, dass es gab einen anderen Funktion, der fehlschlagen könnte wegen dieser Änderung. Aber es ablehnt in meinen Bewusstsein gerufen zuwerden. Ich schlage nur vor, dass wir einige Tests laufen lassen sollen für die Unterschiedliche Fähigkeiten des m65.